code/modules/shuttle/shuttle_creation/shuttle_creator_console.dm

```diff
@@ -12,7 +12,7 @@
 	var/datum/action/innate/shuttle_creator/modify/modify_action = new
 
 /obj/machinery/computer/camera_advanced/shuttle_creator/check_eye(mob/user)
-	if(user.eye_blind || user.incapacitated())
```

Comment:
I don't think this logic should be changed

Reply:
Only removed blind check